source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,128 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions.
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- A domain name pointing to that server
+- Root level access to the server
+
+Installing Mattermost Omnibus
+------------------------------
+
+The Omnibus repositories are configured using a cURL command:
+
+``curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+This command configures the repositories needed for a ``PostgreSQL`` database, an ``nginx`` web server to act as a proxy, and ``certbot`` to issue and renew the SSL certificate.
+
+In order to issue that certificate, the installer requests a domain name and an email address from us. These are used to generate the certificate and deliver any related communications respectively.
```

Comment:
Maybe we should move this line right below the `apt install mattermost-omnibus`, as there two values are going to be requested right after running that command